CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [7.0.0](https://github.com/chanzuckerberg/edu-design-system/compare/v6.0.1...v7.0.0) (2022-11-18)
+
+
+### ⚠ BREAKING CHANGES
+
+* **table:** don't require caption
```

Comment:
nit: can you make this more descriptive? what specific props have been removed or renamed?